psi4/src/psi4/libfock/DirectJK.cc

```diff
@@ -81,8 +94,149 @@ void DirectJK::print_header() const {
         outfile->Printf("    Schwarz Cutoff:    %11.0E\n\n", cutoff_);
     }
 }
-void DirectJK::preiterations() { sieve_ = std::make_shared<ERISieve>(primary_, cutoff_, do_csam_); }
+void DirectJK::preiterations() {
+    sieve_ = std::make_shared<ERISieve>(primary_, cutoff_, do_csam_);
+    
+#ifdef USING_BrianQC
+    if (brianCookie != 0) {
+        double threshold = cutoff_ * (brianCPHFFlag ? 1e-3 : 1e-0); // CPHF needs higher precision
```

Comment:
Are these two only reasonable values? Or is this threshold something we could turn into a runtime option?

Reply:
Due to certain features of our implementation (such as using GPU arithmetic), we usually have to use slightly stricter precision thresholds for Brian's calculations that the ones used by the host. On the other hand, we usually try to make Brian's operation "transparent"; that is, if the user specifies a certain computation with certain precision parameters, then s/he should get the same results with host+Brian as with the host itself. Thus, these multipliers are not normally visible to the user; s/he instead sets the usual parameters like the integral precision threshold, and the required multipliers are determined by us, from several precision measurements.
The two values here are because we observed that the results of a vibrational analysis (using Fock builds for CPHF) are much more precision-sensitive than the single point energy (using Fock builds for SCF). Over time, if the number of calculation types supported by Brian grows, the number of these fitted parameters might increase, and eventually, it might need to be refactored.

Reply:
ah, multipliers, I see now! good, they're already effectively tunable by user.